py/kubeflow/examples/notebook_tests/execute_notebook.py

```diff
@@ -10,7 +10,7 @@
 from kubeflow.testing import util
 
 def prepare_env():
-  subprocess.check_call(["pip3", "install", "-U", "papermill"])
+  subprocess.check_call(["pip3", "install", "-Iv", "papermill==2.0.0"])
```

Comment:
It looks like these changes have been merged.
https://github.com/kubeflow/examples/blob/b8b7179fc253e23455124885beda43399b824b4d/py/kubeflow/examples/notebook_tests/execute_notebook.py#L13